src/main/deltachat/backup.ts

```diff
@@ -15,7 +15,12 @@ import SplitOut from './splitout'
 import { DeltaChatAccount } from '../../shared/shared-types'
 export default class DCBackup extends SplitOut {
   export(dir: string) {
+    this._dc.stopIO()
     this._dc.importExport(C.DC_IMEX_EXPORT_BACKUP, dir, undefined)
+    // look to when it is done
+    this._dc.once('DC_EVENT_IMEX_FILE_WRITTEN', () => {
```

Comment:
by definition, [`DC_EVENT_IMEX_FILE_WRITTEN`](https://c.delta.chat/group__DC__EVENT.html#ga7ffbea55be6a5e6da7ac7e35ba6bf985) does not say that the backup is complete, it just just says that _one_ file is written - so using that as a signal that the backup is complete may cause problems in the future. also, it is not emitted in case of errors, in this case, the app would be continued without io started.

instead, better listen to  [`DC_EVENT_IMEX_PROGRESS(data1=1000)`](https://c.delta.chat/group__DC__EVENT.html#ga006ea41d9c1a76ffc672752484c61e6c) that indicated success by definition. moreover, startIo should also be called if imex fails (indicated by `DC_EVENT_IMEX_PROGRESS(data1=0)`)